test/infra/control/run-tests-isolated.bash

```
@@ -394,7 +394,16 @@ docker run \
                                 # codecov-cli's network_root_folder is the runner's
                                 # /home/runner/work/proxysql/proxysql and the repo content
                                 # lives at <network_root>/proxysql/, so only paths shaped
-                                # as `SF:proxysql/...` get resolved -- the other two forms
+                                # as 'SF:proxysql/...' get resolved -- the other two forms
+                                # (NB: single quotes throughout this comment, never
+                                # backticks. The whole script body is the argument to
+                                # an outer "bash -c" wrapped in double quotes; inside
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)